CONTRIBUTING.md

```diff
@@ -1,35 +1,118 @@
+# Contributing
+
+Thanks for considering to contribute to this project. Please follow the below guidelines before you
+open up a pull request.
+
+## Nomenclature
+
+First a brief intro on some of the terminoligy and concepts we use to make sure the steps below make sense.
+
+_TODO: dmn, sttr, imtr etc. Consider moving this section to README.md instead_
+
+- permit, ('vergunning' in dutch). You might need a permit for an activity (activiteit).
+- topic, a type of doing (werkzaamheid in dutch). Could be multiple activities / permits in one topic. It corresponds with a configured checker with one or more permits.
+- sttr-file; sttr is an XML standard for dutch DSO legislation
+- xml to json; we convert sttr-XML to JSON for better performance
+- sttr-builder; the tool used to build sttr-files
+- check; the activity of checking whether you need a permit (dutch: vergunning)
+- visitor; the person performing a check
+- checker; the tool itself, intro, register lookups, a set of questions and conclusion
```

Comment:
Nice!
Can you change:
checker; the tool itself, intro, register lookups, a set of questions and conclusion

to:
checker; the tool itself (including an intro page, register lookups, a set of questions and a conclusion)